### PR TITLE
EZP-24579 Change URL pattern to call PlatformUI

### PR DESCRIPTION
**Pull request by StephaneDiot**
https://jira.ez.no/browse/EZP-24579

## Description

Changing the url to call, to load the PlatformUI application. It's now "myDomainName"/ez#/ instead of "myDomainName"/shell#/

### CODE REVIEW

**Comment by StephaneDiot** [comment]
ping @dpobel

File: Resources/config/routing.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 shell:
```
Comment (dpobel) [inline Resources/config/routing.yml]: you should change that as well (for consistency)

**Comment by dpobel** [comment]
beside the inline comment, +1

**Comment by mhyndle** [comment]
+1

**Comment by andrerom** [comment]
+1

**Pull request merged**